src/editor.rs

```diff
@@ -494,7 +492,11 @@ impl Editor {
     fn draw_left_padding<T: Display>(&self, buffer: &mut String, val: T) -> Result<(), Error> {
         if self.ln_pad >= 2 {
             // \x1b[38;5;240m: Dark grey color; \u{2502}: pipe "│"
-            write!(buffer, "\x1b[38;5;240m{:>2$} \u{2502}{}", val, RESET_FMT, self.ln_pad - 2)?;
+            if self.use_color {
+                write!(buffer, "\x1b[38;5;240m{:>2$} \u{2502}{}", val, RESET_FMT, self.ln_pad - 2)?;
```

Comment:
Nit: Comment can be moved
```suggestion
            if self.use_color {
                // \x1b[38;5;240m: Dark grey color; \u{2502}: pipe "│"
                write!(buffer, "\x1b[38;5;240m{:>2$} \u{2502}{}", val, RESET_FMT, self.ln_pad - 2)?;
```